### PR TITLE
Header, Main Nav accessibility updates

### PR DESCRIPTION
**Pull request by uqtgeor4**
- Outline to Search toggle button reinstated
- Tabbing through hidden elements of header sorted, tabs through only when visible
- Tabbing through hidden submenu of site header sorted, tabs through only when visible
- Added aria-haspopup="true" to site header menu Level 1 lists that have submenus
- Removed aria-expanded="false" on submenu list items as it isn't required
- Added "aria-label" to both Main & Global navigation <nav>

### CODE REVIEW

**Review by Bcrunk — COMMENTED** [review]
When I use the MAC screen reader the toggle button to open search has the same title as the Search button for submitting the search query. 

<button class="search-toggle__button">Search</button>

I recommend something like "Toggle Search" or a dynamic name "Aria" the swaps between "open/show/toggle Search" and "Close/hide/toggle Search"

**Review by Bcrunk — COMMENTED** [review]
When I navigated the Main Menu with mac Voice over... it read the link titles but did not identify that there was expandable content that could be toggled... there was nothing to distinguish the text links form a ones without children.

**Review by uqnm — COMMENTED** [review]
@Bcrunk @uqtgeor4 could you check if you guys also have the issue with tabbing skipping over the 'Search all UQ websites'?

**Review by uqnm — APPROVED** [review]

File: packages/storybook-html/src/components/header/header.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
                 </div>
```
[the remark this answers is not in the document]
Comment (uqtgeor4) [reply packages/storybook-html/src/components/header/header.html:53]: It skips over the global radio buttons initially to focus on the input field, but once the search area is opened it tabs through the radio buttons and title in the DOM order

**Comment by uqtgeor4** [comment]
> When I use the MAC screen reader the toggle button to open search has the same title as the Search button for submitting the search query.
> 
> `<button class="search-toggle__button">Search</button>`
> 
> I recommend something like "Toggle Search" or a dynamic name "Aria" the swaps between "open/show/toggle Search" and "Close/hide/toggle Search"

File: packages/storybook-html/src/components/header/header.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (uqtgeor4) [reply packages/storybook-html/src/components/header/header.html:53]: It now reads the Search title also as tabindex="0" is applied to it

**Comment by uqtgeor4** [comment]
> @Bcrunk @uqtgeor4 could you check if you guys also have the issue with tabbing skipping over the 'Search all UQ websites'?

It now reads the Search title also as tabindex="0" is applied to it

**Comment by uqtgeor4** [comment]
> Toggle Search



> When I use the MAC screen reader the toggle button to open search has the same title as the Search button for submitting the search query.
> 
> `<button class="search-toggle__button">Search</button>`
> 
> I recommend something like "Toggle Search" or a dynamic name "Aria" the swaps between "open/show/toggle Search" and "Close/hide/toggle Search"

Changed text from "Search" to "Toggle Search"

**Comment by uqtgeor4** [comment]
> When I navigated the Main Menu with mac Voice over... it read the link titles but did not identify that there was expandable content that could be toggled... there was nothing to distinguish the text links form a ones without children.

It distinguishes by saying "Collapsed Menu Popup Link" for menus with submenus & just "Collapsed Link" for one without submenus on MAC VOICEOVER

**Comment by uqtgeor4** [comment]
Removed   aria-expanded="false"  on menu links that didn't have submenu drop-downs

**Pull request merged**